### DISCUSSION TITLE
Comments/suggestions

### DISCUSSION THREAD

**Discussion by XavierHG** [fholger/farcry_vrmod#24] [General]
Although the mod is excellent, I'm just leaving few comments of what I've noticed so far.

- No button assigned to exit _Credits_ from main menu (only keyboard).
- Change or remove the red square hit indicator (as it is today it does not add anything, and the immersion suffers)
- Option to hide hud elements on screen.
- Hand-only is seen with most weapons, and full right arm model with others (screenshot https://prnt.sc/7VywW4OwVxUq)
- The minimap and the detection indicator are not displayed correctly (screenshot https://prnt.sc/7VywW4OwVxUq)
- When you change "_Environment Quality_" in advanced video options, the slider "_Vegetation Render Dist_" in VR options changes to zero automatically. I noticed this because I FIRST set the VR options, and THEN set graphics to maximum, as result all the far trees were gone.

Some wishes...
- I would like VR support for:
  64-bit content
  "_Matto_" series mod (especially 3 and 4)
  "_The Delta Sector_" mod
(These mods in particular adds great single player story-driven missions)

**Comment by fholger** [comment]
I cannot fix the full right arm in code as that is an inconsistency in the weapon models and would have to be changed in the model itself.
What is wrong with the minimap?

64-bit support is impossible, there was no compatible SDK released for the 64-bit patch.

I will not be working on making mods compatible, I just don't have the time or interest. That's something other people will have to do.

**Comment by XavierHG** [comment]
OK, the minimap is stretched, I watched some videos of your mod and it doesn't happen, so it must be my system, nothing really important.

**Comment by XavierHG** [comment]
If it's not complex to explain, could you tell me what to change to run faster just putting the weapon down, without needing to separate my hands? (I play with both hands on the weapons all the time)
In moments of action when you need fast switch between run-walk and shoot, results uncomfortable to make these moves, added to the limitation of stamina that there is by default.

By the way, amazing job with version 0.7.0a, draw distance is now insane, snipers can be seen from miles away.

**Comment by terracide** [comment]
Not sure where the best place is to report bugs, but the game tends to crash for me when attempting to use most mounted turrets.  Otherwise, this is one amazing mod, I'm having a LOT of fun with this. perfection!

**Comment by Kentypoo** [comment]
Thank you and we love you fholger! ;)  I play in my 2 car garage and use my space as much as possible. However My chaperone in this mod keeps getting so dim I barely see it or I can't see it at all. I think the Seated Universe Type game style in SteamVR that the mod uses is the cause. So I am hopeing this can be updated to roomscale style. I tend to play close to my chaperone, and Steam now auto dims it if you are close for long and it doesn't to come back to full visibility when moving back to the center of the playspace at least not in this mod. OVRAS doesn't let me fix the floor in what they call a Seated Universe Type game style. That is where I am getting the term. I've seen this in a few other games but not many.  I also use manul redirected walking via OVRAS and this might not play nice with the Steated Universe Type Chaperone. 
        I was ducking in real life behind boxes in the training mission and I was still getting shot, doesn't seem like my hit box is ducking with me. The mod also pauses my stick movement if I am also walking in my garage while trying to use smooth stick movement locomotion. I love how the mod lets me walk up in game slopes in roomscale but when walking down them in roomscale I have to walk like 10 feet into the air before I fall back to the games ground. Anyways, I'm having fun with the mod but we really gotta at least figure out this chaperone dimming thing.  Thanks again. you rock!

**Comment by fholger** [comment]
Yes, the tracking space is still seated, and yes, there is no support for real-life crouching. This is also mentioned in the [manual](https://farcryvr.de/manual/).
Changing the tracking space is in the works, but it is an awkward process because you suddenly have to deal with a load of edge cases and extra checks you need to do to make sure the camera ends up in the right place in all situations. That's why I left this change for last.

As for roomscale movement, I'm afraid chances are this is going to stay as it is. There is just no good way to translate roomscale movement into the game, and the current implementation is the best tradeoff I have found.

**Reply by Kentypoo** [reply]
oops, I missed the vr manual. I saw it pop up and thought it was the old manual for the flat game. So I just figured out my chaperone keeps disapearing on me cuz im using manual redirected walking via OVRAS's space turn, and if I use your stick turn instead for manual redirected walking, my chaperone stays visible.  I prefer OVRAS's space turn, but not enough to stop playing ;P.  I'd love a hologram/red dot sight on the assualt rifle, or even just some glow in the dark paint on the iron sights in game to make aiming down the sights without the cursor more fun. Im not really a fan of the one hand weapon bobbing but at least you can cure it by double gripping. My protube is working great in this mod, just gotta remember to double grip (squeeze) that front hand! Thanks again! It's even more fun now that I know to stop using OVRAS's space turn. have a great day!

**Reply by fholger** [reply]
I'm afraid I can't do anything about the sights or the animations - that would require changes to the actual weapon models, and that's out of my abilities.
